libs/sdk-backend-mockingbird/src/recordedBackend/types.ts

```diff
@@ -46,6 +47,16 @@ export type RecordedBackendConfig = IAnalyticalBackendConfig & {
      * Default: 'uri'
      */
     useRefType?: RecordedRefType;
+
+    /**
+     * Specify validator that returns boolean for provided URL value and validation context type.
+     */
+    securitySettingsUrlValidator?: SecuritySettingsUrlValidator;
+
+    /**
+     * Specify function that builds organization scope from organization ID.
```

Comment:
please mention what happens by default when this prop is not specified

Reply:
Fixed.